app/src/main/java/com/tomtom/sdk/examples/maps/mapdetails/MapDetailItemBase.kt

```diff
@@ -0,0 +1,7 @@
+package com.tomtom.sdk.examples.maps.mapdetails
+
+open class MapDetailItemBase(
```

Comment:
Is this parent class really needed, I don't see usage of this type. If it is it would be better to make it an interface